content/Minecraft-Server-Hosting.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: article
+title: Install Minecraft Server
+description: 
+  So you want some of that blocky goodness? Well, get Java and some graphics cards and let's go!
+keywords:
+  - Minecraft
+  - Mojang
+  - Games
+hidden: false
+section: software
+---
+
+# Hosting A Minecraft Server
+
+Minecraft 1.17 dropped on June 8, 2021. In this article, we will be going over how to set up a Minecraft server on your Pop!_OS or Ubuntu install. A Minecraft System76 recommends having a minimum of 8Gb of RAM but for a good experience, 16Gb of RAM is recommended. Any hard drive over 128GB for a basic server but if you want to backups 1TB to 2TB is recommended. This article will live mostly in the terminal, if you want a refresher on the terminal you can see our article [here](https://support.system76.com/articles/terminal-basics).
```

Comment:
Typo: "A Minecraft System76 recommends"

Should be: "System76 OR Mojang recommends..."

Reply:
8GB minimum.

16GB "recommended" or "preferred."

Reply:
Any hard drive 128GB for a basic server... <- sentence fragment.

Reply:
This article will use mostly Terminal commands.

Reply:
Corrected spelling/grammar